File: JavaScript/Array 2D y 3D/Orden por columnas de matriz 2D.js
const matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

console.log("---Recorrido por columnas---");
for (let j = 0; j < matriz[0].length; j++) {
    let columna = "Columna " + j + ": ";
    for (let i = 0; i < matriz.length; i++) {
        columna += matriz[i][j] + " ";
    }
    console.log(columna);
}

File: Python/Array 2D y 3D/Orden por columnas de matriz 2D.py
matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

print("---Recorrido por columnas---")
for j in range(len(matriz[0])):
    print("Columna", j, ":", end=" ")
    for i in range(len(matriz)):
        print(matriz[i][j], end=" ")
    print()